api/config/dev.exs

```diff
@@ -20,7 +20,7 @@ config :api, Api.Repo,
 config :api, ApiWeb.Endpoint,
   # Binding to loopback ipv4 address prevents access from other machines.
   # Change to `ip: {0, 0, 0, 0}` to allow access from other machines.
-  http: [ip: {127, 0, 0, 1}, port: 4000],
+  http: [ip: {0, 0, 0, 0}, port: 4000],
```

Comment:
want to be able to access the endpoints from local into the container.

Reply:
This isn't the environment where that happens. You want to update the `prod` environment, which is where all deployments run (even if it's in the staging or other environment according to the MBTA).